Authentication/AuthenticationBootstrapper.swift

```diff
@@ -63,10 +64,10 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
         }
     }
 
-    /**
+    /*
         Bootstraps your project with the authentication framework,
```

Comment:
What about something like this?

```
Bootstraps your application with the authentication framework,
starting with the login screen if no user is logged in.
Otherwise, it runs your application's Main View Controller.
```